### PR TITLE
Add slack management

### PR DESCRIPTION
**Pull request by skaex**

### CODE REVIEW

File: lib/tasks/sync_docusign_to_salesforce.rake [hunk 1]
```diff
@@ -8,6 +8,15 @@ namespace :sync do
...
     tasks.salesforce_api = RowanBot::SalesforceAPI.new
-    tasks.sync_signed_waivers_to_salesforce(days)
+    emails = tasks.sync_signed_waivers_to_salesforce(days)
+
+    # I'll tag these tasks here since they depend on emails of people who
+    # recently signed
+    unless emails.empty?
+      tasks.assign_slack_to_users(emails)
+      tasks.assign_peer_groups_to_users(emails)
+      tasks.assign_to_peer_group_channel_in_slack(emails, ['abdulmajid.hamza@bebraven.org', 'aleks.calderon@bebraven.org'])
```
Comment (sadleb) [inline lib/tasks/sync_docusign_to_salesforce.rake:18]: Can we make the admin array an ENV var?  E.g. SLACK_ADMIN_EMAILS

File: examples/booster/sign_waivers.rb [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
 
 tasks = RowanBot::Tasks.new
 tasks.docusign_api = RowanBot::DocuSignAPI.new
 tasks.salesforce_api = RowanBot::SalesforceAPI.new
+tasks.slack_api = RowanBot::SlackAPI.new
 emails = tasks.sync_signed_waivers_to_salesforce(30)
-p emails
+tasks.assign_slack_to_users(emails)
+tasks.assign_peer_groups_to_users(emails)
+tasks.assign_to_peer_group_channel_in_slack(emails, ['abdulmajid.hamza@bebraven.org'])
```
Comment (sadleb) [inline examples/booster/sign_waivers.rb:12]: Use same ENV var as I propose in the comment below for admin emails?

File: lib/rowan_bot/salesforce_api.rb [hunk 3]
```diff
@@ -27,46 +29,92 @@ def assign_peer_groups_to_program(program_id, cohort_size)
...
     private
 
     attr_reader :client
 
+    def map_email_with_peer_group(email)
+      participant = find_participant_by_email(email)
+      { email: email, peer_group: participant.Cohort__r.Name }
+    end
+
+    def assign_peer_groups_to_user_email(email, max_cap = 10)
```
Comment (sadleb) [inline lib/rowan_bot/salesforce_api.rb:53]: Will we need to change max_cap? If so, we should allow the rake task to take it as an argument, like we did for days of DocuSign envelopes to process.

Comment (skaex) [reply lib/rowan_bot/salesforce_api.rb:53]: I think the requirement on Max 10 was clear. I just did this incase we'll want to change it later.

**Comment by sadleb** [comment]
We can't really test this on Heroku without merging and pushing. We setup the following two buildpacks for selenium to work in the app though:
- https://github.com/heroku/heroku-buildpack-google-chrome
- https://github.com/heroku/heroku-buildpack-chromedriver

**Pull request merged**